components/ILIAS/Test/classes/class.ilTestSkillLevelThresholdsGUI.php

```diff
@@ -86,20 +89,20 @@ public function getTestId(): int
 
     private function saveSkillThresholdsCmd(): void
     {
-        if (strtolower($_SERVER['REQUEST_METHOD']) == 'post') {
+        if (strtolower($_SERVER['REQUEST_METHOD']) === 'post') {
```

Comment:
To further encapsulate global access, we could consider implementing an `isMethod(string $method)` in the `RequestDataCollector`. 

What are your thoughts on this approach, @kergomard?